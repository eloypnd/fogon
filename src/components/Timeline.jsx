import React from 'react'
import { Container, Flex } from 'rebass'
import { TimelineItem } from '.'

const Timeline = ({posts}) => (
  <Container>
    {posts && (
      <Flex flexWrap='wrap'>
        {posts.map(
          item => <TimelineItem key={item.id} {...item} />
        )}
      </Flex>
    )}
  </Container>
)

export default Timeline
