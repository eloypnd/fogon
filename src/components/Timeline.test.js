import React from 'react'
import renderer from 'react-test-renderer'
import Timeline from './Timeline'

it('Timeline renders correctly', () => {
  const tree = renderer
    .create(<Timeline />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
