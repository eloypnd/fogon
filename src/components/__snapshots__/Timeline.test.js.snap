// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timeline renders correctly 1`] = `
<div
  className="sc-eHgmQL xPffg sc-bdVaJa iHZvIS"
/>
`;
